### DISCUSSION TITLE
Explain the Meaning of Variable

### DISCUSSION THREAD

**Discussion by dpklear** [katspaugh/wavesurfer.js#3353] [Q&A] [answered]
Hello,

If someone can give description of variables defined on the silence JS

const minValue = 0.01
  const minSilenceDuration = 0.1
  const mergeDuration = 0.2

thankyou

**Comment by katspaugh** [comment, marked as answer]
`minValue` = minimum loudness (from -1 to 1) considered non-silent
`minSilenceDuration` = ignore periods of silence below this value, in seconds
`mergeDuration` = the threshold length (in seconds) of non-silent segements that should be merged into one larger non-silent segment, e.g. if a non-silent segment is below 0.2 seconds, it will be merged with the next one.
